nms/app/packages/magmalte/app/views/traffic/ApnOverview.js

```diff
@@ -146,7 +145,18 @@ export default function ApnOverview() {
               {title: 'Qos Profile', field: 'qosProfile', type: 'numeric'},
               {title: 'Added', field: 'added', type: 'datetime'},
             ]}
-            menuItems={[{name: 'Edit'}, {name: 'Deactivate'}, {name: 'Remove'}]}
+            handleCurrRow={(row: ApnRowType) => setCurrRow(row)}
+            menuItems={[
+              {name: 'Edit'},
```

Comment:
since we don't support edit at this point, we might as well get rid of it and rename 'edit json' -> 'edit'.